packages/@react-stately/radio/src/useRadioGroupState.ts

```diff
@@ -11,21 +11,30 @@
  */
 
 import {RadioGroupProps} from '@react-types/radio';
+import React, {useState} from 'react';
 import {useControlledState} from '@react-stately/utils';
 
 export interface RadioGroupState {
   selectedRadio: string | undefined,
-  setSelectedRadio: (value: string) => void
+  setSelectedRadio: (value: string) => void,
+  focusableRadio: string | undefined,
+  setFocusableRadio: (value: string) => void,
 }
 
-export function useRadioGroupState(props: RadioGroupProps):RadioGroupState  {
+export function useRadioGroupState(props: RadioGroupProps): RadioGroupState {
   let [selectedRadio, setSelected] = useControlledState(props.value, props.defaultValue, props.onChange);
 
+  let activeChildren = React.Children.toArray(props.children).filter(child => !child.props.isDisabled);
+  let defaultFocusableRadio = selectedRadio || activeChildren[0] && activeChildren[0].props.value;
+
+  let [focusableRadio, setFocusableRadio] = useState(defaultFocusableRadio);
```

Comment:
I think we should leave it as `null` by default (when there's no selected radio), until one of the radios receives focus. All of the the radios would have `tabIndex={0}` by default. This way, if you shift tabbed into the radio group from after, the last radio would be focused instead of the first.